Secondlesson/OperationsOnImages/InfoPixel.cpp

```diff
@@ -0,0 +1,23 @@
+//
+// Created by s1000633 on 5/17/21.
+//
+
+#include "InfoPixel.h"
+
+InfoPixel::InfoPixel(cv::Mat &image, int x, int y) {
```

Comment:
if image passed by reference doesn't suppose to be changed, pass it by const reference